code/index.d.ts

```diff
@@ -1,26 +1,28 @@
-// Type definitions for code 4.0.0
+// Type definitions for code 4.0
 // Project: https://github.com/hapijs/code
 // Definitions by: Prashant Tiwari <https://github.com/prashaantt>
 // Definitions: https://github.com/DefinitelyTyped/DefinitelyTyped
 
 /** Generates an assertion object. */
-export function expect<T>(value: T | T[], prefix?: string): AssertionChain<T>;
+declare function expect<T>(value: T | T[], prefix?: string): AssertionChain<T>;
```

Comment:
We don't hold back from exporting interfaces in case someone wants to use them. A return type isn't an internal implementation detail.